jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/InvoiceResource.java

```diff
@@ -999,8 +1000,9 @@ public Response getTags(@PathParam(ID_PARAM_NAME) final UUID invoiceId,
                             @QueryParam(QUERY_AUDIT) @DefaultValue("NONE") final AuditMode auditMode,
                             @javax.ws.rs.core.Context final HttpServletRequest request) throws TagDefinitionApiException, InvoiceApiException {
         final TenantContext tenantContext = context.createTenantContextNoAccountId(request);
-        final Invoice invoice = invoiceApi.getInvoice(invoiceId, tenantContext);
-        return super.getTags(invoice.getAccountId(), invoiceId, auditMode, includedDeleted, tenantContext);
+        // See https://github.com/killbill/killbill/issues/1273
+        final UUID accountId = AuditLevel.NONE.equals(auditMode.getLevel()) ? null : invoiceApi.getInvoice(invoiceId, tenantContext).getAccountId();
+        return super.getTags(accountId, invoiceId, auditMode, includedDeleted, tenantContext);
```

Comment:
Yeah, the change makes sense from a perf point of view - handling this Nullable `accountId` is not great though, but what to do...